docs/guide/installation.md

```diff
@@ -78,11 +78,22 @@ poetry run lnbits-cli --help
 
 ```sh
 cd lnbits
-# Stop LNbits with `ctrl + x`
-git pull
-# Keep your poetry install up to date, this can be done with `poetry self update`
+# Stop LNbits with `ctrl + x` or with service manager
+# sudo systemctl stop lnbits
+
+# Update LNbits
+git pull --rebase
+
+# To keep your poetry up to date, use
+poetry env use python3.12
+poetry env remove python3.9
+poetry env list
```

Comment:
Maybe this should be listed a separate prerequisite (not to be misunderstood that it has to be ran each time)

Reply:
Rephrased instructions